src/app/my-profile/components/my-profile/my-profile.component.scss

```diff
@@ -234,17 +241,20 @@
     flex-direction: column;
     align-items: center;
     gap: 10px;
-    margin-top: 80px;
+    margin-top: 40px;
 
     h2 {
       @include fonts.font__effra--bold;
       font-size: 36px;
+      @media only screen and (max-width: 480px) {
+        font-size: 24px;
+      }
```

Comment:
Wir sollten uns meiner Meinung nach auf einen Ansatz einigen - `max-width` (= desktop first) oder `min-width` (= mobile first).

Reply:
Guter Punkt.
Dachte auch, es gäbe eine Vorgabe und hatte zuvor den Code durchsucht. Tatsächlich tritt beides auf, `min-with` allerdings häufiger. An dieser Stelle klar `min-with`, da es im file auch sonst durchgehend verwendet wird. Habe es geändert.
Sollte aber auch in anderen Komponenten harmonisiert werden.

Reply:
Genau, ich würde das Stück für Stück gerade ziehen und dann in dem Zuge auch Breakpoints als zentrale Variablen definieren.
Da gibt es sicherlich noch einige Punkte die man schöner gestalten kann.

Für den Moment finde ich aber `min-width` auch schöner 👍

Reply:
Die Breakpoint Variables (in _variables.scss) gibt's ja schon, hab sie bei der Korrektur verwendet.